.github/workflows/build-and-test.yml

```diff
@@ -40,3 +40,25 @@ jobs:
 
     - name: mergesat build
       run: make d r sh lr ld lsh
+
+  WebAssembly:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Install Packages
+      run: sudo apt-get install make
+
+    - uses: actions/checkout@v1
+
+    - uses: actions/cache@v2
+      with:
+        path: emsdk-cache
+        key: emsdk-${{ runner.os }}
+
+    - uses: mymindstorm/setup-emsdk@v9
+      with:
+        actions-cache-folder: emsdk-cache
+
+    - name: mergesat build
+      run: make CXX=em++ CXXFLAGS="-Werror -Wno-unused-parameter" lr ld lp
```

Comment:
Please do not add -Werror here. There are other warnings in the code (e.g. signed comparison) that are not handled properly at the moment. As soon as em++ catches up warning on those, this would break. Thanks.